src/commctl/client_script.py

```diff
@@ -255,7 +255,7 @@ def create_host(self, address, **kwargs):
         infile = kwargs['ssh-priv-key']
         b64_bytes = base64.b64encode(infile.read())
         data['ssh_priv_key'] = b64_bytes.decode()
-        if 'cluster' in kwargs:
+        if kwargs['cluster'] is not None:
```

Comment:
Sorry, I just now looked at this.

The `'cluster'` argument is optional, so I think this is going to throw a `KeyError` if it's not there.

Maybe instead: `if kwargs.get('cluster') is not None:`